src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Todo.css';
+
+const Task = (props) => {
+  return (
+  <div 
+    className={props.task.completed && 'complete' } 
```

Comment:
So this returns a warning in React Tools. Part of the warning also tells us how to fix it.

`...If you used to conditionally omit it with className={condition && value}, pass className={condition ? value : undefined} instead.`